src/app/recipes/recipes.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-recipes',
+  templateUrl: './recipes.component.html',
+  styleUrls: ['./recipes.component.scss']
+})
+export class RecipesComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
If you don't use `OnInit` you can remove it and the lifecycle-method `ngOnInit()` too.